### PR TITLE
Social media sharing

### PR DESCRIPTION
**Pull request by bondarik1995**

### CODE REVIEW

File: app/communities-config.js [hunk 1]
```diff
@@ -117,8 +117,8 @@ const communitiesConfig = {
   dev: {
     1: {
       origin: 'http://localhost:3000',
-      src: TelosLogo,
-      styles,
+      // src: TelosLogo,
```
Comment (sergeyi23) [inline app/communities-config.js:120]: What is the reason for commenting this?

File: app/containers/ViewQuestion/ContentHeader.jsx [hunk 2]
```diff
@@ -146,6 +205,69 @@ const ContentHeader = props => {
             <FormattedMessage {...messages.voteToDelete} />
           </Button>
 
+          {questionData.user === userInfo.user && (
```
Comment (sergeyi23) [inline app/containers/ViewQuestion/ContentHeader.jsx:208]: Are we allowing users to share only own question?
Users should be allowed to share any question that they like.

File: app/containers/ViewQuestion/ContentHeader.jsx [hunk 3]
```diff
@@ -146,6 +205,69 @@ const ContentHeader = props => {
...
+            <DropdownBox>
```
Comment (sergeyi23) [inline app/containers/ViewQuestion/ContentHeader.jsx:209]: Sharing dropdown needs to be a separate component.

File: app/containers/ViewQuestion/ContentHeader.jsx (hunk 2 above)
Comment (bondarik1995) [reply app/containers/ViewQuestion/ContentHeader.jsx:208]: This component is used for question and answer. Sharing is used only for questions by this condition and user can share any question

File: app/communities-config.js (hunk 1 above)
Comment (bondarik1995) [reply app/communities-config.js:120]: I commented this to have current site version not community version

File: app/containers/ViewQuestion/ContentHeader.jsx (hunk 3 above)
Comment (bondarik1995) [reply app/containers/ViewQuestion/ContentHeader.jsx:161]: added separate component

**Pull request merged**